### PR TITLE
try code

### PR DESCRIPTION
**Pull request by tanishchugh01**

### CODE REVIEW

File: Tanish/armstrongnumber.cpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#include<iostream>
+#include<math.h>
+using namespace std;
+
+int main()
+{
+    int n,dig=0,sum=0;
+    cin>>n;
+    for(int i=n;i>0;i/=10)
```
Comment (BudhirajaMadhav) [inline Tanish/armstrongnumber.cpp:10]: Add a line before every fore loop

File: Tanish/armstrongnumber.cpp [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        dig++;
+    for(int i=n;i>0;i/=10)
+        sum+=pow(i%10,dig);
+    cout<<sum<<endl;
+    if(sum==n)
+        cout<<"YES";
+    else
+        cout<<"NO";
+    return 0;
```
Comment (BudhirajaMadhav) [inline Tanish/armstrongnumber.cpp:23]: Add a line before return statement

File: Tanish/armstrongnumber.cpp [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+int main()
+{
+    int n,dig=0,sum=0;
+    cin>>n;
+    for(int i=n;i>0;i/=10)
+        dig++;
+    for(int i=n;i>0;i/=10)
+        sum+=pow(i%10,dig);
+    cout<<sum<<endl;
+    if(sum==n)
```
Comment (BudhirajaMadhav) [inline Tanish/armstrongnumber.cpp:18]: Add a line before conditional statements

**Review by BudhirajaMadhav — CHANGES_REQUESTED** [review]
Amend the mentioned changes

File: Tanish/armstrongnumber.cpp (hunk 3 above)
Comment (tanishchugh01) [reply Tanish/armstrongnumber.cpp:18]: resolved this issue

**Review by BudhirajaMadhav — APPROVED** [review]

**Pull request merged**